stix/idl/processing/spectrogram/stx_fsw_sd_spectrogram2ospex.pro

```diff
@@ -125,15 +128,31 @@ function stx_fsw_sd_spectrogram2ospex, spectrogram, specpar = specpar, time_shif
     srmfilename =  fits_info_params.srmfile
 
     fits_info_params.grid_factor = grid_factor
+    fits_info_params.detused = detector_label + ', Pixels: ' + pixel_label
+
+    if keyword_set(xspec) then begin
+      ;xspec in gneral works with energy depandent systematic errors
+      e_axis = spectrum_in.e_axis
+      n_energies = n_elements(e_axis.mean)
+      sys_err  = fltarr(n_energies)
+
+      idx_below10kev = where(e_axis.mean lt 10, cb10)
+      sys_err[*] = 0.03
+      if cb10 gt 0 then sys_err[idx_below10kev] = 0.05
+      idx_below7kev = where(e_axis.mean lt 7, cb7)
+      if cb7 gt 0 then sys_err[idx_below7kev] = 0.07
```

Comment:
So we have different systematic error if different energy ranges?

Reply:
Yes, this is currently only relevant for XSPEC as OSPEX can only handle a single value but the idea is to capture our confidence in the low energy response compared with the higher energies.